site/source/user-manual/service-guides/matrix/index.rst

```diff
@@ -0,0 +1,14 @@
+.. _synapse-service:
```

Comment:
As the "Matrix Index," I think this should be labeled as such, then files in this directory can be for services/clients, i.e. Synapse, Conduit, etc